### PR TITLE
feat(kleros-liquid-emails): make Draw message more accurate

### PR DESCRIPTION
**Pull request by epiqueras**
Closes kleros/court#21

### CODE REVIEW

File: src/event-handlers/kleros-liquid-emails.js [hunk 1]
```diff
@@ -45,15 +45,19 @@ const handlers = {
...
+                ? timeAgo.format(
+                    (Number(dispute.lastPeriodChange) +
+                      Number(
+                        (await klerosLiquid.methods
+                          .getSubcourt(dispute.subcourtID)
+                          .call()).timesPerPeriod[0]
+                      )) *
+                      1000
+                  )
+                : 'as soon as all other jurors are drawn'
```
Comment (satello) [inline src/event-handlers/kleros-liquid-emails.js:59]: "Voting starts once all jurors have been selected" is more laymen's

Comment (epiqueras) [reply src/event-handlers/kleros-liquid-emails.js:59]: "Select" kind of implies someone deciding.

We use the term "Draw" everywhere.

Comment (satello) [reply src/event-handlers/kleros-liquid-emails.js:59]: I don't see Select that way but it is fine if we use Draw other places as well.

Comment (epiqueras) [reply src/event-handlers/kleros-liquid-emails.js:59]: Yeah, consistency is more important here.

**Review by satello — APPROVED** [review]

**Pull request merged**